blog.html

```diff
@@ -22,6 +22,98 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_380_release_announcement"></a>
+                    <a href="#apache_kafka_380_release_announcement">Apache Kafka 3.8.0 Release Announcement</a>
+                </h2>
+                xx July 2024 - Josep Prat (<a href="https://twitter.com/jlprat">@jlprat</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.8.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.8.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/documentation.html#upgrade_3_8_0">Upgrading to 3.8.0 from any version 0.8.x through 3.7.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>
+                    In a previous release, 3.6,
+                    <a href="https://kafka.apache.org/38/documentation.html#tiered_storage">tiered storage</a> was released as early availability feature.
+                    In this release, Tiered Storage now supports clusters configured with multiple log directories (i.e. JBOD feature). This feature still remains as early availability.
+                </p>
+                <p>
+                    In the last release, 3.7, <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-858%3A+Handle+JBOD+broker+disk+failure+in+KRaft">KIP-858</a>
+                    was released in early availability. Since this version, JBOD in KRaft is considered production ready.
```

Comment:
The term used in the 3.7 release was _early access_. Would it be clearer to keep the same term?

What do you think of this variation?

```suggestion
                    <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-858%3A+Handle+JBOD+broker+disk+failure+in+KRaft">KIP-858</a> introduced JBOD support in KRaft in the 3.7 release. Since then, limitations have been addressed and it is no longer an early access feature.
```